src/GitHub/Data/Options.hs

```diff
@@ -491,16 +495,70 @@ issueFilter f = IssueMod $ \opts ->
 -- Issues repo
 -------------------------------------------------------------------------------
 
+-- | See <https://docs.github.com/en/rest/reference/issues#list-repository-issues>.
+--   Retrieved: 2021-12-08
+--
+-- Parameters of "list repository issues" (@get /repos/{owner}/{repo}/issues@)
```

Comment:
Oh please don't copy the GitHub docs. That's why there is a link. It's a maintenance burden without an easy way to check if something had changed.